app/javascript/components/Results/Results.tsx

```diff
@@ -158,10 +166,16 @@ export const Results = ({ query = '', results = null, additionalResults = null,
             />
           } */}
 
+          {/* Results: Images */}
+          {imagesResults.length > 0 && <ImagesPage images={imagesResults}/>}
+          
           {/* Results */}
           {results && results.length > 0 ? 
             <> 
               {results.map((result, index) => {
+                if (result.image) {
```

Comment:
We are using the GridContainer (https://github.com/GSA/search-gov/pull/1342/files#diff-1cb55c059efac4ba4ed9b9695bbb241dee5a3885a9ff1af03bb0b47f66189f96R30) as a parent wrapper for individual items like Jobs, RSS, etc. But for Images, we need to put all the images in this GridContainer for it to follow the 4-column UI for which I am putting all the Image elements with col-3 classes in the above GridContainer wrapper.

We can however put the images without skipping the results map but it will require a major refractor across all other components we have, but in my opinion I think we should keep it like this only.